cryptofeed/callback.py

```diff
@@ -25,8 +25,16 @@ async def __call__(self, *args, **kwargs):
 
 
 class TradeCallback(Callback):
-    async def __call__(self, *, feed: str, pair: str, side: str, amount: Decimal, price: Decimal, order_id=None, timestamp: float, receipt_timestamp: float):
-        await super().__call__(feed, pair, order_id, timestamp, side, amount, price, receipt_timestamp)
+
+    def __init__(self, callback, include_order_type=False):
+        self.include_order_type = include_order_type
```

Comment:
I see, rather than have it enabled by default, its user configured. Interesting, I wonder if it would be advantageous to have it always on vs user enabled. Makes me also wonder about other fields, maybe they should all be configurable. Things to think about - I think this is good for now, I'll think about the ramifications of some of those other thoughts and let you know if I'm leaning, in the future, to something that deviates from this significantly. Only other thought is to have an exclude list, thats normally empty for all CBs but for trades the default is include order type, which then would require the user to un-exclude it. idk. I tend to come up with my best ideas when I'm away from the computer so will think about this.

Reply:
I think for simplicity always-on would be best, but I was wary of breaking already-defined callbacks out in the wild. If there ends up being more fields to add in the future, perhaps wrapping all of the args in a `Trade` object or similar would simplify things so that new fields can be added without breaking the function signature (other than the one-time break to go towards a single arg rather than the list)